core/src/main/java/org/frankframework/http/rest/ApiListenerServlet.java

```diff
@@ -524,6 +522,11 @@ else if(segment.startsWith("{") && segment.endsWith("}")) {
 				 */
 				Message result = listener.processRequest(body, messageContext);
 
+				if (listener.getMethod().equals(ApiListener.HttpMethod.HEAD)) {
```

Comment:
Laurens has a PR to allow the ApiListener to handle multiple HTTP methods. Then this will no longer work.
To be better prepared for that better to check the method of the actual HTTP Request itself, not the Listener configuration.

But in addition I'm also not sure that this is the right thing to do. We have code to set content-type further down. I would keep all code for setting content-type together.